src/win/pty.cc

```diff
@@ -131,6 +131,23 @@ void throw_winpty_error(const char *generalMsg, winpty_error_ptr_t error_ptr) {
   winpty_error_free(error_ptr);
 }
 
+static NAN_METHOD(PtyGetExitCode) {
+  Nan::HandleScope scope;
+
+  if (info.Length() != 1
+    || !info[0]->IsNumber()) // pidHandle
+  {
+    Nan::ThrowError("Usage: pty.getExitCode(pidHandle)");
+    return;
+  }
+
+  uint32_t pidHandle = info[0]->Uint32Value();
+  DWORD exitCode = 0;
+  GetExitCodeProcess((HANDLE)pidHandle, &exitCode);
```

Comment:
@Tyriar Btw I think the 64 --> 32 bit conversion is fine and in line with the docs as it preserves the lower 32 bits.